examples/jsm/renderers/webgpu/WebGPUUniform.js

```diff
@@ -37,12 +37,12 @@ class FloatUniform extends WebGPUUniform {
 		this.boundary = 4;
 		this.itemSize = 1;
 
-		Object.defineProperty( this, 'isFloatUniform', { value: true } );
-
 	}
 
 }
 
+FloatUniform.prototype.isVector4Uniform = true;
```

Comment:
The flags in this file are completely wrong...

Reply:
Friendly reminder: https://github.com/mrdoob/three.js/pull/20100#issuecomment-780479068